__tests__/app.test.js

```diff
@@ -41,3 +41,49 @@ describe("GET/api/topics", () => {
       });
   });
 });
+
+describe("GET/api/articles", () => {
+  it("Responds with an error if path is not found", () => {
+    return request(app)
+      .get("/anything")
+      .expect(404)
+      .then(({ body }) => {
+        console.log(body);
+        expect(body.msg).toBe("Path not found");
+      });
+  });
+
+  it("Responds with an array from the articles data", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Array);
+        expect(articles.length).toBeGreaterThan(0);
```

Comment:
Look at the data so you can get an exact number on the length of the array you're expecting to get back.